frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -323,14 +323,45 @@ function ReaderHighlight:genHighlightDrawerMenu()
             callback = function()
                 self.view.highlight.disabled = not self.view.highlight.disabled
             end,
-            hold_callback = function(touchmenu_instance)
+            hold_callback = function()
                 self:toggleDefault()
             end,
             separator = true,
         },
         get_highlight_style("lighten"),
         get_highlight_style("underscore"),
         get_highlight_style("invert"),
+        {
+            text_func = function()
+                return T(_("Lightening factor: %1"), G_reader_settings:readSetting("highlight_lighten_factor", 0.2))
```

Comment:
confusing name

Reply:
True, but haven't really found anything better :/.

Reply:
`Highlight contrast`, maybe?

Reply:
(It's grayed out in other modes, so it doesn't really need to match/contain "lighten" to make it clear that applies to the ligthen mode).

Reply:
Highlight opacity? (as "the higher the darker")

Reply:
Works for me, yeah ;).

Reply:
Yes, I suppose that'll do.